Jenkinsfile

```diff
@@ -19,7 +19,16 @@ void buildOnYoctoNode(String variant, String image) {
 }
 
 def variantMap = [:]
-def variantList = env.VARIANT_LIST.split()
+def variantList = []
+
+try {
+    variantList = env.JOB_VARIANT_LIST.split()
+} catch(e) {
+    println(e.getMessage())
+    echo "Using the default \"VARIANT_LIST\" for build"
```

Comment:
Maybe specifying here (in a short way such as `rpi-qtauto-dev`) which variants we are building would be better.

Reply:
I don't understand this comment. Do you mean that the long string is hard to read? Should I have the default values specified in a comment?

Reply:
echo "Using \"rpi-qtauto-dev\" and \"intel-qtauto-dev\" for build"

Reply:
Ok I get it. Yes I will add it.